src/DssSpell.sol

```diff
@@ -55,139 +47,39 @@ contract DssSpellAction is DssAction {
-        // Poll: https://vote.makerdao.com/polling/QmSovaxn
-        // NOTE: Skip on goerli
+        // ---------- Immunefi CU MKR Vesting Transfer ----------
+        // Immunefi CU - 6.34 MKR - 0xd1F2eEf8576736C1EbA36920B957cd2aF07280F4
+        // Forum: https://forum.makerdao.com/t/mip39c3-sp13-removing-is-001/22392
+        // MIP: https://mips.makerdao.com/mips/details/MIP40c3SP41#sentence-summary
+        // Skip on goerli
 
-        // JanSky | 2023-10-01 00:00:00 to 2024-09-30 23:59:59 | 504,000.00 DAI | 0xf3F868534FAD48EF5a228Fe78669cf242745a755
-        // VoteWizard | 2023-10-01 00:00:00 to 2024-09-30 23:59:59 | 504,000.00 DAI | 0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1
-        // JanSky | 2023-10-01 00:00:00 to 2024-09-30 23:59:59 | 216.00 MKR | 0xf3F868534FAD48EF5a228Fe78669cf242745a755
-        // VoteWizard | 2023-10-01 00:00:00 to 2024-09-30 23:59:59 | 216.00 MKR | 0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1
+        // ---------- Housekeeping - GUSD & USDP - Add Jar & Conduit Contracts to Chainlog ----------
+        // Forum: https://forum.makerdao.com/t/proposed-housekeeping-items-upcoming-executive-spell-2023-11-01/22477
 
+        // Add `RwaJar` at 0xf2E7a5B83525c3017383dEEd19Bb05Fe34a62C27 as MCD_PSM_GUSD_A_JAR
+        // Skip on goerli
 
-        // ---------- BA Labs MKR Distribution  ----------
-        // Forum: https://forum.makerdao.com/t/mip40c3-sp25-risk-core-unit-mkr-compensation-risk-001/9788
-        // Poll: https://vote.makerdao.com/polling/QmUAXKm4
-        // NOTE: Skip on goerli
+        // Add `RwaSwapInputOutputConduit2` at 0x6934218d8B3E9ffCABEE8cd80F4c1C4167Afa638 as MCD_PSM_GUSD_A_INPUT_CONDUIT_JAR
+        // Skip on goerli
 
-        // BA Labs - 175 MKR - 0x5d67d5B1fC7EF4bfF31967bE2D2d7b9323c1521c
+        // Add `RwaJar` at 0x8bF8b5C58bb57Ee9C97D0FEA773eeE042B10a787 as MCD_PSM_PAX_A_JAR
+        // Skip on goerli
 
-        // ---------- AVC Member Compensation  ----------
-        // Forum: https://forum.makerdao.com/t/avc-member-participation-rewards-q3-2023/22349
-        // Poll: https://vote.makerdao.com/polling/QmSovaxn#poll-detail
-        // NOTE: Skip on goerli
+        // Add `RwaSwapInputConduit2` at 0xDa276Ab5F1505965e0B6cD1B6da2A18CcBB29515 as MCD_PSM_PAX_A_INPUT_CONDUIT_JAR
+        // Skip on goerli
 
-        // opensky - 20.85 MKR - 0x8e67ee3bbeb1743dc63093af493f67c3c23c6f04
-        // DAI-Vinci - 12.51 MKR - 0x9ee47F0f82F1A6F45C4E1D25Ce95C321D8C8356a
-        // IamMeeoh - 20.85 MKR - 0x47f7A5d8D27f259582097E1eE59a07a816982AE9
-        // ACRE DAOs - 20.85 MKR - 0xBF9226345F601150F64Ea4fEaAE7E40530763cbd
-        // Harmony - 20.85 MKR - 0xE20A2e231215e9b7Aa308463F1A7490b2ECE55D3
-        // Res - 20.85 MKR - 0x8c5c8d76372954922400e4654AF7694e158AB784
-        // seedlatam.eth - 20.85 MKR - 0x0087a081a9b430fd8f688c6ac5dd24421bfb060d
+        // Bump chainlog as it has been modified.
+        DssExecLib.setChangelogVersion("1.17.1");
```

Comment:
The comment to this action is a false statement. For better readability, you can explain why are you doing it, despite there were no changes in this spell
```diff
-        // Bump chainlog as it has been modified.
+        // Bump chainlog version for consistency, as it will be modified on mainnet

```